README.md

```diff
@@ -41,11 +41,12 @@ Packages required:
   * Pytest
   * Dotenv
   * Selenium
+  * Gunicorn
 ```
 
-To update any missing dependencies:
+To update any missing or new dependencies:
 ```bash
-$ poetry add --dev <package_name>
+$ poetry add <package_name>
 ```
 
 ## Trello API
```

Comment:
I'd add a note saying that you need the right todo list names in this section of the `README.md` (else the application will fail)